R/bootstrap.R

```diff
@@ -197,7 +204,7 @@ renv_bootstrap_download_cran_latest <- function(version) {
   type  <- spec$type
   repos <- spec$repos
 
-  catf("* Downloading renv %s ... ", version, appendLF = FALSE)
+  catf("* Downloading %s from CRAN ... ", type, appendLF = FALSE)
```

Comment:
Do you think that's worth fixing? It's going to make the message less informative for the 95% of folks using the default.

Reply:
I would prefer to, since we don't report the repository URL anywhere else.

Would it make sense to log the repository used for downloading `renv` earlier on instead? So you could have something like:

```
* Using repository 'https://cloud.r-project.org'
* Downloading binary ... OK
< ... >
```

Otherwise, can we please drop "from CRAN"?

Reply:
For the non-archive case (which I think is most common) we can show the URL instead of CRAN:

```
* Downloading binary from <https://cran.rstudio.com> ... OK
```